meta-lxatac-bsp/recipes-bsp/barebox/files/lxatac/env/nv/autoboot_abort_key

```diff
@@ -0,0 +1 @@
+ctrl-c
```

Comment:
The noise on the UART should be resolved by correct the pull settings in the fixup mentioned earlier.

Reply:
The UART should be unaffected, shouldn't it? It's the button pullups that have changed.
I think we could keep this change. It should make the boot more robust to all kinds of interference.

Reply:
Yes, you're right.